### ISSUE TITLE
As a user I want to archive a mail

### ISSUE THREAD

**Issue by fabienmoyon** [OpenPaaS-Suite/esn-frontend-inbox#177]
#### EXPECTED

a very nice to have is the archive button at the same level that trash and read/nonread (confy for gmail users, it means most people)

**Comment by MichaelBailly** [comment]
What does "archive email" means ?

**Comment by alagane** [comment]
It means copying them in archive directory.

Why you would use it?

* To keep your main mailbox clean
* To avoid remove policy removing mails older than 2 months for example in the main mailbox, they are not removed on archives
* To find easily important mail you are keeping

On Thunderbird, there is an Archives directory, with a subdirectory for each year (I think you can configure it), and archiving a mail automatically places it in the right directory.

**Comment by chibenwa** [comment]
It means **moving** them in archive directory, no?

**Comment by alagane** [comment]
> It means **moving** them in archive directory, no?
> 
> > To avoid remove policy removing mails older than 2 months for example in the main mailbox, they are not removed on archives
> 
> We have no such things yet, right?

Yes it was moving and not copying, I corrected my message. And I did this because I could set up cleaning in Thunderbird for example.
